demos/mqtt/mqtt_demo_mutual_auth/demo_config.h

```diff
@@ -112,6 +112,37 @@
  * #define CLIENT_PRIVATE_KEY_PATH    "...insert here..."
  */
 
+/**
+ * @brief The username string, required to authenticate to your MQTT broker.
```

Comment:
The sentence could provide misleading meaning that username is _required_ for client authentication. It can be updated to mention something like "The username value for authenticating client to MQTT broker when username/password based client authentication is used."

Similar suggestion for the `CLIENT_PASSWORD` macro @brief

Reply:
Agree. Thanks